src/test/python/deep_qa_test/layers/noisy_or_test.py

```diff
@@ -0,0 +1,45 @@
+from unittest import TestCase
+import numpy as np
+from numpy.testing import assert_array_almost_equal
+
+from keras import backend as K
+from keras.layers import Input, Masking
+from keras.models import Model
+from deep_qa.layers.noisy_or import NoisyOr, BetweenZeroAndOne
+
+class TestNoisyOr(TestCase):
+    # pylint: disable=no-self-use
+    def test_general_case(self):
+
+        input_layer = Input(shape=(3,2,), dtype='float32', name="input")
+        axis = 2
+        noisy_or_layer = NoisyOr(axis=axis)
+        output = noisy_or_layer(input_layer)
+        model = Model([input_layer], output)
+
+        # Testing general unmasked case.
```

Comment:
I think this is a subset of the batched case, and you don't really need to have this test.  The batched version is good enough.